src/main/webapp/resources/js/pages/projects/samples-metadata-import2/components/SampleMetadataImportFileUploader.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import {
+  Alert,
+  Divider,
+  message,
```

Comment:
user notification not message

Reply:
Please see 32f8108f30132873eda492d9a3cb0832c6e2638a.